presto-kafka/src/test/java/io/prestosql/plugin/kafka/encoder/json/TestISO8601JsonDateTimeFormatter.java

```diff
@@ -0,0 +1,102 @@
+import org.testng.annotations.Test;
+
+import java.time.ZoneOffset;
+import java.util.Optional;
+
+import static io.prestosql.spi.type.TimeZoneKey.UTC_KEY;
+import static io.prestosql.testing.DateTimeTestingUtils.sqlDateOf;
+import static io.prestosql.testing.DateTimeTestingUtils.sqlTimeOf;
+import static io.prestosql.testing.DateTimeTestingUtils.sqlTimeWithTimeZoneOf;
+import static io.prestosql.testing.DateTimeTestingUtils.sqlTimestampOf;
+import static io.prestosql.testing.DateTimeTestingUtils.sqlTimestampWithTimeZoneOf;
+import static io.prestosql.testing.assertions.Assert.assertEquals;
+
+public class TestISO8601JsonDateTimeFormatter
+{
+    private static final DateTimeFormat DATE_TIME_FORMAT = DateTimeFormat.ISO8601;
+
+    private void testDate(SqlDate value, String expectedLiteral)
+    {
+        String actualLiteral = DATE_TIME_FORMAT.getFormatter(Optional.empty()).formatDate(value);
+        assertEquals(actualLiteral, expectedLiteral);
+    }
+
+    private void testTime(SqlTime value, int precision, String expectedLiteral)
+    {
+        String actualLiteral = DATE_TIME_FORMAT.getFormatter(Optional.empty()).formatTime(value, precision);
+        assertEquals(actualLiteral, expectedLiteral);
+    }
+
+    private void testTimeWithTZ(SqlTimeWithTimeZone value, String expectedLiteral)
+    {
+        String actualLiteral = DATE_TIME_FORMAT.getFormatter(Optional.empty()).formatTimeWithZone(value);
+        assertEquals(actualLiteral, expectedLiteral);
+    }
+
+    private void testTimestamp(SqlTimestamp value, String expectedLiteral)
+    {
+        String actualLiteral = DATE_TIME_FORMAT.getFormatter(Optional.empty()).formatTimestamp(value);
+        assertEquals(actualLiteral, expectedLiteral);
+    }
+
+    private void testTimestampWithTZ(SqlTimestampWithTimeZone value, String expectedLiteral)
+    {
+        String actualLiteral = DATE_TIME_FORMAT.getFormatter(Optional.empty()).formatTimestampWithZone(value);
+        assertEquals(actualLiteral, expectedLiteral);
+    }
+
+    @Test
+    public void testISO8601DateTimeFunctions()
+    {
+        testDate(sqlDateOf(2020, 8, 14), "2020-08-14");
+        testDate(sqlDateOf(1970, 1, 1), "1970-01-01");
+        testDate(sqlDateOf(1900, 1, 1), "1900-01-01");
+        testDate(sqlDateOf(3001, 1, 1), "3001-01-01");
+
+        testTime(sqlTimeOf(3, 15, 36, 25, 123000000), 3, "15:36:25.123");
+        testTime(sqlTimeOf(3, 15, 36, 25, 0), 3, "15:36:25");
+        testTime(sqlTimeOf(3, 8, 12, 45, 987000000), 3, "08:12:45.987");
+        testTime(sqlTimeOf(3, 0, 0, 0, 0), 3, "00:00");
+        testTime(sqlTimeOf(3, 23, 59, 59, 999000000), 3, "23:59:59.999");
+
+        testTimeWithTZ(sqlTimeWithTimeZoneOf(10, 23, 35, 123000000, ZoneOffset.UTC), "10:23:35.123Z");
+        testTimeWithTZ(sqlTimeWithTimeZoneOf(0, 0, 0, 0, ZoneOffset.UTC), "00:00:00Z");
+        testTimeWithTZ(sqlTimeWithTimeZoneOf(10, 23, 35, 123000000, ZoneOffset.ofHours(2)), "10:23:35.123Z");
```

Comment:
why expected value uses UTC (`Z`)
ISO8601 allows for timezone encoding - don't we want to support it?